src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -600,7 +600,10 @@ export class BrowsingContextImpl {
       throw new InvalidArgumentException(`Invalid URL: ${url}`);
     }
 
-    await this.targetUnblocked();
+    const result = await this.targetUnblocked();
```

Comment:
If we do this everywhere, is it more appropriate to either (i) add a boolean argument to `targetUnblocked` to optionally `throw` on error, OR (ii) always throw unconditionally?